src/index.html

```diff
@@ -1,13 +1,301 @@
+                    Lights integrated into the frame give you always-on visibility day and night.
+                  </p>
+                </div>
+              </article>
+
+              <article class="detail">
+                <div class="detail__photos">
+                  <a href="#" class="detail__link detail__link--square">
+                    <img
+                      class="detail__photo"
+                      src="images/ditails-handle.png"
+                      alt="Hydraulic disc brakes & Lightweight"
+                    >
+                  </a>
+                  <a href="#" class="detail__link detail__link--wide">
+                    <img
+                      class="detail__photo"
+                      src="images/details-last.png"
+                      alt="Hydraulic disc brakes & Lightweight"
+                    >
+                  </a>
+                </div>
+                <h3 class="detail__title">Hydraulic disc brakes & Lightweight</h3>
+                <div class="detail__description">
+                  <p class="detail__description--text">
+                  Brakes with total stopping power the second you make
+                  contact. The removable battery has up to 70km battery
+                   autonomy and weighs only 2.4 kg. Lights integrated
+                   into the frame give you always-on visibility day and night.
+                  </p>
+                </div>
+              </article>
+              <a href="#home" class="detail__button">
+                <button class="button detail__button--text" id="televisions">Explore</button>
+              </a>
+            </div>
+
+          </section>
+
+          <section class="contact-us" id="contact-us">
+            <h2 class="section-title">Contact us</h2>
+            <div class="contacts__content">
+
+              <form
+                action="POST"
+                class="contacts__form"
+                onsubmit="this.reset(); return false;"
+              >
+                <input
+                  type="text"
+                  placeholder="Name"
+                  class="contacts__form__field contacts__form__field--input"
+                  required
+                >
+
+                <input
+                  type="email"
+                  placeholder="Email"
+                  class="contacts__form__field contacts__form__field--input"
+                  required
+                >
+
+                <textarea
+                  name="message"
+                  id="#"
+                  cols="30"
+                  rows="10"
+                  placeholder="Message"
+                  class="contacts__form__field contacts__form__field--textArea"
+                  required
+                ></textarea>
+
+                <button type="submit" class="contacts__form__button button">Send</button>
+              </form>
+              <div class="contacts__contacts-info contacts-info">
```

Comment:
your hover effect looks bad on this links, just change their color, 4'th point in checklist